<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-328P50G" failures="1" tests="2" name="test login" time="26.989" errors="0" timestamp="27 May 2023 20:20:06 GMT">
  <testcase classname="tests.TestLogin" name="loginWithInValidCredentials" time="5.277"/>
  <testcase classname="tests.TestLogin" name="loginWithValidCredentials" time="0.504">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at tests.TestLogin.loginWithValidCredentials(TestLogin.java:39)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginWithValidCredentials -->
</testsuite> <!-- test login -->
